src/app/dashboard/revalidation-history/revalidation-history.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, OnInit } from "@angular/core";
+import {
+  animate,
+  state,
+  style,
+  transition,
+  trigger
+} from "@angular/animations";
+import { MatBottomSheet } from "@angular/material/bottom-sheet";
+import { RevalidationNotesComponent } from "../revalidation-notes/revalidation-notes.component";
+import { MatDialog } from "@angular/material/dialog";
+import { RevalidationFormComponent } from "../revalidation-form/revalidation-form.component";
+
+@Component({
+  selector: "app-revalidation-history",
+  templateUrl: "./revalidation-history.component.html",
+  styleUrls: ["./revalidation-history.component.scss"],
+  animations: [
+    trigger("detailExpand", [
+      state("collapsed", style({ height: "0px", minHeight: "0" })),
+      state("expanded", style({ height: "*" })),
+      transition(
+        "expanded <=> collapsed",
+        animate("225ms cubic-bezier(0.4, 0.0, 0.2, 1)")
+      )
+    ])
+  ]
+})
+export class RevalidationHistoryComponent implements OnInit {
+  dataSource: PeriodicElement[];
+  watchTrainee: boolean;
+  columnsToDisplay = [
+    "episodeId",
+    "recommendation",
+    "outcome",
+    "gmcSubDueDate",
+    "ActSubDate",
+    "submittedBy"
+  ];
+  expandedElement: PeriodicElement | null;
+  constructor(private bottomSheet: MatBottomSheet, public dialog: MatDialog) {}
+
+  ngOnInit(): void {
+    this.dataSource = [
```

Comment:
as before this is probs one for later but this mock data should really live in state management